python/lsst/afw/image/multiband.py

```diff
@@ -0,0 +1,1001 @@
+
+        Returns
+        -------
+        singles: tuple
+            Tuple of `MaskedImage` objects for each band,
+            where the `image`, `mask`, and `variance` of each `single`
+            point to the multiband objects.
+        """
+        singles = []
+        if image is None:
+            image = self.image
+        if mask is None:
+            mask = self.mask
+        if variance is None:
+            variance = self.variance
+
+        dtype = image.array.dtype
+        for f in self.filters:
+            maskedImage = MaskedImage(image=image[f], mask=mask[f], variance=variance[f], dtype=dtype)
+            single = Exposure(maskedImage, dtype=dtype)
+            singles.append(single)
+        return tuple(singles)
+
+    @classmethod
+    def fromButler(cls, butler, filters, filterKwargs, *args, **kwargs):
+        """Load a multiband exposure from a butler
+
+        Because each band is stored in a separate exposure file,
+        this method can be used to load all of the exposures for
+        a given set of bands
+
+        Parameters
+        ----------
+        butler: `Butler`
+            Butler connection to use to load the single band
+            calibrated images
+        filters: list or str
+            List of filter names for each band
+        filterKwargs: dict
+            Keyword arguments to pass to the Butler
+            that are different for each filter.
+            The keys are the names of the arguments and the values
+            should also be dictionaries, with filter names as keys
+            and the value of the argument for the given filter as values.
+        args: list
+            Arguments to the Butler.
+        kwargs: dict
+            Keyword arguments to pass to the Butler
+            that are the same in all bands.
+
+        Returns
+        -------
+        result: `MultibandExposure`
+            The new `MultibandExposure` created by combining all of the
+            single band exposures.
+        """
+        # Load the Exposure in each band
+        exposures = []
+        for f in filters:
+            if filterKwargs is not None:
+                for key, value in filterKwargs:
+                    kwargs[key] = value[f]
+            exposures.append(butler.get(*args, filter=f, **kwargs))
+        return cls(filters, exposures)
+
+    def computePsfKernelImage(self, coords=None):
+        """Get a multiband PSF image
+
+        The PSF Kernel Image is computed for each band
+        and combined into a (filter, y, x) array and stored
+        as `self._psfImage`.
+        The result is not cached, so if the same PSF is expected
+        to be used multiple times it is a good idea to store the
+        result in another variable.
+
+        Parameters
+        ----------
+        coords: `Point2D` or `tuple`
+            Coordinates to evaluate the PSF. If `coords` is `None`
+            then `Psf.getAveragePosition()` is used.
```

Comment:
We tend to use "coord" to mean sky coordinates; it'd be a bit more in keeping withe precedent and a bit less confusing to stack users to call this "position".